### ISSUE TITLE
Guild removes characters upon logout

### ISSUE THREAD

**Issue by Schnieble** [Atlantiss/NetherwingBugtracker#4255]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: After creating a guild characters are removed from the guild upon logging out.  All ranks including GM are prone, thus eventually deleting the guild entirely once everyone has logged off or there are no ranking officers capable of re-inviting.  Upon logging back in on your character the guild tab is grey but you can still read gchat, but not write gchat.  We have tried to create 3 different guilds now, handing the charter in once in IF and twice in SW

First picture contains guild chat from my POV (GM); current status as guildless
![wowscrnshot_011519_222346](https://user-images.githubusercontent.com/46739139/51231146-54410400-1917-11e9-96bb-0391f742beca.jpg)
Second picture contains guild members POV stating I am current GM although on my screen i am guildless
![inguildevidence](https://user-images.githubusercontent.com/46739139/51231074-117f2c00-1917-11e9-928a-c108b7e12096.jpg)

**Current behaviour**:

**Expected behaviour**: 

**Server Revision**:

**Comment by Schnieble** [comment]
I was able to resolve this problem and create a guild this morning somehow after multiple attempts by multiple people over the past week.  After getting signatures I logged out at the Darnassus inn and deleted both my WDB and my WTF folders. I then continued to log back on and hand in the charter to create the guild. I kicked everyone from the guild, changed a few  parameters, created a new rank and logged back off. Upon logging in the guild is still existent.
